tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -13,7 +13,8 @@
 
 class Homescreen(Base):
 
-    name = 'Default Home Screen'
+    name = 'Homescreen'
```

Comment:
'Default Home Screen' is new name of the homescreen. See: https://github.com/mozilla-b2g/gaia/blob/master/apps/verticalhome/manifest.webapp#L2